src/sources/amqp.rs

```diff
@@ -135,7 +136,14 @@ impl SourceConfig for AmqpSourceConfig {
         let schema_definition = self
             .decoding
             .schema_definition(log_namespace)
-            .with_standard_vector_source_metadata();
+            .with_standard_vector_source_metadata()
+            .with_source_metadata(
```

Comment:
This isn't the fluent source... Should we put this in a separate pr?

Reply:
He had noticed I had incorrectly not included this type def in the amqp and aws_s3 fixes I PR'd earlier. I don't have a problem including it in this PR, but if there's a group preference for separating 👍

Reply:
Ditto to Spencer's response.
We had chatted offline about this as I observed this missing from the schema def in this PR (after it was merged in) https://github.com/vectordotdev/vector/pull/15205

The relevant lines where that field is added are here :
https://github.com/vectordotdev/vector/blob/6099e0d3eace517d1a6b2cda5daab61c70a89ac2/src/sources/amqp.rs#L234-L255

If you feel strongly about it being pulled out, I can do that.

Main reason I didn't was just that it's a pretty small change. In hindsight I should have added a note about it to the PR description.

Reply:
It's not a huge issue. It's just I had also added this to the amqp source, but haven't submitted a PR for it yet so one of us will get a conflict.. but not really a big deal.